tests/tests_unit/test_api/test_files.py

```diff
@@ -126,6 +146,12 @@ def download_link_callback(request):
 
 
 class TestFilesAPI:
+    def test_create(self, mock_file_create_response):
+        file_metadata, upload_url = FILES_API.create(name="bla")
+        response_body = mock_file_create_response.calls[0].response.json()
+        assert FileMetadata._load(response_body) == file_metadata
+        assert response_body["uploadUrl"] == upload_url
```

Comment:
Set expected values on the left side of the operator.

Reply:
That is what I'm doing here.

Reply:
ah of course 🤦‍♂